CMakeLists.txt

```diff
@@ -208,7 +217,6 @@ add_library(rotors_gazebo_imu_plugin SHARED src/gazebo_imu_plugin.cpp)
 add_library(gazebo_opticalFlow_plugin SHARED src/gazebo_opticalFlow_plugin.cpp)
 target_link_libraries(gazebo_opticalFlow_plugin ${OpticalFlow_LIBS})
 add_library(gazebo_lidar_plugin SHARED src/gazebo_lidar_plugin.cpp)
-add_library(gazebo_irlock_plugin SHARED src/gazebo_irlock_plugin.cpp)
```

Comment:
@siddhu45 You continue to not having the latest master from upstream. Please isse `git pull https://github.com/PX4/sitl_gazebo master` on you local repo, commit the changes and push again.